Automattic/wp-calypso
# Videos UI: Add a header component for onboarding.

This PR adds a header component for the videos UI, as originally designed for the onboarding hero flow. Since the modal footer component we have in place for My Home is identical in need for onboarding, we'll just use that (possibly confusing when looking at file names, but let's see how it goes).

| Mobile | Desktop |
| --- | --- |
| <img width="330" alt="Screen Shot 2021-11-25 at 10 50 49 AM" src="https://user-images.githubusercontent.com/349751/143490324-5bcc30f7-f811-4e34-881e-fce1d53a5229.png"> | <img width="1100" alt="Screen Shot 2021-11-25 at 10 49 42 AM" src="https://user-images.githubusercontent.com/349751/143490328-b9377cf7-ad00-413c-b912-7fff2f16dd56.png"> |

| Mobile (Course Complete) | Desktop (Course Complete) |
| --- | --- |
| <img width="375" alt="Screen Shot 2021-12-01 at 2 49 17 PM" src="https://user-images.githubusercontent.com/349751/144326855-0c4c72b6-bc0c-4553-831f-a83c9970cd72.png"> | <img width="1623" alt="Screen Shot 2021-12-01 at 2 48 45 PM" src="https://user-images.githubusercontent.com/349751/144326866-d20cb205-65ee-4e3e-abf2-99cd04bf527a.png"> |

**Testing Instructions**
* First verify that the current "modal" components still look and behave correctly on My Home.
* Load the new onboarding components by altering those pulled in by `<BloggingQuickStartModal />` (see below).
* Verify the elements and behaviour match with the original [Figma](EuqfSnWpfYx8fgiBlVmbuA-fi-2119%3A1111).
* Also check the "course complete" view with the button; this can be triggered by setting the `isCourseComplete` prop to `true` in `modal-footer-bar.jsx`.

<img width="650" alt="Screen Shot 2021-12-01 at 2 45 56 PM" src="https://user-images.githubusercontent.com/349751/144326475-0a52d0ca-fa03-4577-9210-3f160e1a0518.png">
